packages/terra-heading/src/terra-dev-site/doc/example/HeadingVariations.jsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
 import Heading from 'terra-heading/lib/Heading';
+import classNames from 'classnames/bind';
+import styles from './colors.module.scss';
+
+const cx = classNames.bind(styles);
 
 const HeadingVariations = () => (
   <div>
-    <Heading level={3} color="#304FFE" isItalic>
+    <Heading level={3} className={cx(['neutral'])} isItalic>
 H3 heading with color set to
       <code>rgb(46,125,50)</code>
```

Comment:
```suggestion
      {' '}
      <code>'info' className</code>&nbsp;
```

nit - could you also make sure to add any missing space `{' '}` before and after each `<code>` elements in this example group.

Reply:
Fixed in https://github.com/cerner/terra-core/pull/2211/commits/618ae6178edc4a734ef31a044c3bc8d570051b5c